pkg/healthcheck/internal_test.go

```diff
@@ -34,5 +37,30 @@ func TestHttpCall(t *testing.T) {
 
 	resp, err := http.Get(server.URL + "/")
 	require.NoError(t, err)
-	assert.Equal(t, http.StatusNoContent, resp.StatusCode)
+	assert.Equal(t, http.StatusOK, resp.StatusCode)
+
+	hr := getHealthCheckResponse(t, resp, )
+
+	assert.Equal(t, "up", hr.StatusMsg)
+	assert.Equal(t, hc.upTimeStats.UpTime, hr.upTimeStats.UpTime)
+	assert.True(t, hc.upTimeStats.StartedAt.Equal(hr.upTimeStats.StartedAt))
+
+	time.Sleep(300)
```

Comment:
this will add an extra 5 minutes for each test run which is overkill

Reply:
Those are `Nanosecond`. I wanted to check the case where some time's passed and the service went down, that the uptime does not get incremented.